docs/api/modules/store.md

```diff
@@ -0,0 +1,182 @@
+| `waveform`                    | `boolean`	    | True if the image has waveform data, for ecg                              |
+| `viewport`                    | `viewportType`| Type of the viewport.                                                     |
+
+The `Viewport` type includes the following key properties:
+
+| Property          | Type          | Description                                                                   |
+|-------------------|---------------|-------------------------------------------------------------------------------|
+| `scale`	        | `number`	                                    | Current scale factor for the viewport.        |
+| `rotation`	    | `number`                                      | Current rotation for the viewport.            |
+| `translation`	    | `{x:number, y:number}`	                    | Current translation for the viewport.         |
+| `voi`	            | `{windowCenter:number, windowWidth:number}`	| Current windowing settings for the viewport.  |
+| `default`         | `viewportType`                                | Default viewport settings.                    |
+
+### Initialization
+
+The store must be initialized before use:
+
+```typescript
+import { store } from 'larvitar';
+store.initialize();
+```
+
+### Viewport Management
+
+A Viewport is a window that displays an image. The store manages viewport-specific data, such as the slice and time positions, pixel spacing, dimensions, and modality. It also stores viewer settings like zoom, rotation, and translation.
+
+To add a viewport to the store, use the `addViewport` function with the HTML element ID where the viewport is rendered:
+
+```typescript
+import { store } from 'larvitar';
+store.addViewport(elementId);
+```
+
+To get the viewport data for a specific element ID, use the `get` function with the viewport key:
+
+```typescript
+import { store } from 'larvitar';
+const viewport = store.get(["viewports", elementId]);
+```
+
+To delete a viewport from the store, use the `deleteViewport` function with the viewport key:
+
+```typescript
+import { store } from 'larvitar';
+store.deleteViewport(elementId);
+```
+
+### Series Management
+
+The store tracks series data, including image identifiers, caching status, and loading progress. 
+
+To add a series to the store, use the `addSeriesId` function with the series UID and imageIds:
+
+```typescript
+import { store } from 'larvitar';
+store.addSeriesId(seriesUID, imageIds);
+```
+
+To remove a series from the store, use the `removeSeriesId` function with the series UID:
+
+```typescript
+import { store } from 'larvitar';
+store.removeSeriesId(seriesUID);
+```
+
+To remove all series from the store, use the `resetSeriesIds` function:
+
+```typescript
+import { store } from 'larvitar';
+store.resetSeriesIds();
+```
+
+### Useful functions
+
+The store provides several utility functions for managing data:
+
+| Function              | Parameters                    | Description  |
```

Comment:
For these calls, I would explain which effect they have on the viewport (change slice rendered, ...)

Reply:
They do not have any effect on the rendered viewport